### ISSUE TITLE
Patroni db mode uses source database "template1"  [ URGENT ]

### ISSUE THREAD

**Issue by muhabbad** [cybertec-postgresql/pgwatch2#613]
I use Patroni db type in my instances.yaml file. And my pgwatch collector uses template1 database. In documentation, it is stated that :

> NB! All “continuous” modes expect access to “template1” or “postgres” databasess of the specified cluster to determine the database names residing there.

In my case, i don't use continuous mode in my configuration. What should i do? Is this expected behavior in Patroni mode?

**Comment by muhabbad** [comment]
This is urgent for me. Is there someone to help me?

**Comment by pashagolub** [comment]
Of course, please check support options here: https://www.cybertec-postgresql.com/en/services/postgresql-support/standard/product-support/
